### PR TITLE
feat(l10n):add-l10n-config

### PR DESCRIPTION
**Pull request by muhAzri**
Add Localization Feature

### CODE REVIEW

File: lib/l10n/app_bg.arb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "помогнете c лапа",
```
Comment (BurningAXE) [inline lib/l10n/app_bg.arb]: Let's keep the English app name in all languages.

**Review by BurningAXE — CHANGES_REQUESTED** [review]
Good job! 
Only a small change request to keep the app name consistent across languages.

File: lib/l10n/app_bg.arb (hunk 1 above)
Comment (muhAzri) [reply lib/l10n/app_bg.arb]: Just Fix that, btw can i get the full strings of bulgarian so i can define all that on the arb so others volunteers just need to use it

File: lib/l10n/app_bg.arb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "Help A Paw",
```
Comment (BurningAXE) [inline lib/l10n/app_bg.arb]: Hey, sorry about misunderstanding - I mean we can hardcode the name as it's not going to change in any language.

File: lib/presentation/home_screen/home_screen.dart [hunk 3]
```diff
@@ -21,27 +21,30 @@ class _HomeScreenState extends State<HomeScreen> {
...
       ),
-      body: GoogleMap(
-        onMapCreated: _onMapCreated,
-        initialCameraPosition: CameraPosition(
-          target: _center,
-          zoom: 11.0,
-        ),
-        zoomControlsEnabled: false,
-        myLocationEnabled: true,
+      body: Stack(
```
Comment (BurningAXE) [inline lib/presentation/home_screen/home_screen.dart]: We can also probably get rid of the Stack if it will contain a single child!?

**Review by BurningAXE — CHANGES_REQUESTED** [review]

File: lib/l10n/app_bg.arb (hunk 2 above)
Comment (muhAzri) [reply lib/l10n/app_bg.arb]: Oh okay copy that so i will leave the .arb empty for now ?

File: lib/presentation/home_screen/home_screen.dart (hunk 3 above)
Comment (muhAzri) [reply lib/presentation/home_screen/home_screen.dart]: Okay On It

File: lib/app.dart
Comment (ehsankkk1) [inline lib/app.dart:1]: I think Easy localizations package might be a better option especially for Arabic language  in the future

File: lib/presentation/faq/faq.dart
Comment (ehsankkk1) [inline lib/presentation/faq/faq.dart:1]: merge you previouse branch with this one please

**Review by ehsankkk1 — COMMENTED** [review]
Merge your previous branch with this branch please

**Review by ehsankkk1 — CHANGES_REQUESTED** [review]
Merge your previous branch with this branch please

**Review by ehsankkk1 — CHANGES_REQUESTED** [review]
Merge your previous branch with this branch please

**Review by ehsankkk1 — CHANGES_REQUESTED** [review]
Merge your previous branch with this branch please

**Review by ehsankkk1 — APPROVED** [review]
All good thanks

**Pull request closed without merging**